README.md

```diff
@@ -117,3 +117,75 @@ npm publish --access public
 git push
 git push --tags
 ```
+
+
+### Packaging with vercel.pkg
+There are some nuances when packaging with native modues.
+Included native modules are level witch include leveldown and utp-native.
+
+If a module is not set to public then pkg defaults to including it as bytecode.
+To avoid this breaking with the `--no-bytecode` flag we need to add `--public-packages "*"`
+
+#### leveldown
+To get leveldown to work with pkg we need to include the prebuilds with the executable.
+after building with pkg you need to copy from `node_modules/leveldown/prebuilds` -> `path_to_executable/prebuilds`
+You only need to include the prebuild for the arch you are targeting. eg for linux-x64 you need `prebuild/linux-x64`.
+
+The folder structure for the executable should look like this.
+- linux_executable_elf
+- prebuilds
+  - linux-x64
+    - (node files)
+
+#### utp-native
+Including utp-native is simpler, you just need to add it as an asset for pkg.
+Add the following lines to the package.json.
+```json
+"pkg": {
+    "assets": "node_modules/utp-native/**/*"
+  }
+```
+
+#### threads.js
+To make sure that the worker threads work properly you need to include the compiled worker scripts as an asset.
+This can be fixed by adding the following to `package.json`
+```json
+"pkg": {
+    "assets": "dist/bin/worker.js"
+  }
+```
+
+If you need to include multiple assets then add them as an array.
+
+```json
+"pkg": {
+    "assets": [
+      "node_modules/utp-native/**/*",
+      "dist/bin/worker.js"
+    ]
+  }
+```
+
+#### working with nix-build
```

Comment:
This should be: `### Native Module Toolchain`.

This section actually not part of `Releasing`.

It should be it's own larger section that comes before `Docs Generation`.